### DISCUSSION TITLE
Adding Nepali Language in HA

### DISCUSSION THREAD

**Discussion by Rrp13** [home-assistant/frontend#20128] [Localization]
English Name: Nepali
Native Name: नेपाली

**Comment by bramkragten** [comment]
What is your username on Lokalise?

**Reply by Rrp13** [reply]
My username is Rrp13.

**Reply by bramkragten** [reply]
You should be able to translate now on Lokalise

**Reply by Rrp13** [reply]
Thank You for adding the language.
